### PR TITLE
Don't report name after forcerename for namelocked users

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

File: server/chat.js [hunk 1]
```diff
@@ -1984,10 +1984,10 @@ Chat.monitors = {};
 /** @type {Map<string, string>} */
 Chat.namefilterwhitelist = new Map();
 /**
- * Inappropriate userid : forcerenaming staff member's userid
- * @type {Map<string, string>}
+ * userids that have been previously forcerenamed
+ * @type {Set<string>}
  */
-Chat.forceRenames = new Map();
+Chat.forceRenames = new Set();
```
Comment (Zarel) [inline server/chat.js]: There's no need not to track these just because they're unused.

**Comment by Zarel** [comment]
It looks like this removes the mention of the staff member who did the forcerename? Why?

**Comment by whalemer** [comment]
The mention was removed a while back because it was annoying and you can just check modlog if you need to know. If you want me to still keep the data around then I can, I guess, but afaik it's not used anywhere.

**Comment by Zarel** [comment]
I'd prefer for the data to still be kept around (if only to make it easier to hotpatch).

**Pull request merged**